src/components/Provider.svelte

```diff
@@ -1,61 +1,84 @@
 <script lang="ts">
   import {
     dotApi,
-    storeProviderId,
     storeConnected,
```

Comment:
This file is almost all bug fixing because it was somewhat blocking the stuff I was working on.